src/cmds/scripts/post_install.py

```diff
@@ -0,0 +1,463 @@
+				sys.exit(11)
+			else:
+				print "Comm service registered"
+	# Register Server and create server database with default values
+	with open(os.devnull, 'w') as fs:
+			theproc= subprocess.Popen(["pbs_account.exe", "--unreg", ServerPath], stdout=fs, stderr=fs )
+			theproc.communicate()
+	with open(os.devnull, 'w') as fs:
+			theproc= subprocess.Popen(["pbs_account.exe", "--reg", ServerPath, "-a", username, "-p", password], stdout=fs, stderr=fs)
+			theproc.communicate()
+			if theproc.returncode > 0:
+				print "Failed to register Scheduler"
+				sys.exit(11)
+			else:
+				print "Server service registered"
+	CreateServerdbFile()
+	#Register and Start Mom
+	with open(os.devnull, 'w') as fs:
+			theproc= subprocess.Popen(["pbs_account.exe", "--unreg", MomPath], stdout=fs, stderr=fs )
+			theproc.communicate()
+	with open(os.devnull, 'w') as fs:
+			theproc= subprocess.Popen(["pbs_account.exe", "--reg", MomPath, "-a", username, "-p", password], stdout=fs, stderr=fs)
+			theproc.communicate()
+			if theproc.returncode > 0:
+				print "Failed to register Mom"
+				sys.exit(11)
+			else:
+				print "Mom service registered"
+	with open(os.devnull, 'w') as fd:
+			theproc= subprocess.Popen(["sc", "start", "pbs_mom"], stdout=fd, stderr=fd)
+			theproc.communicate()
+			if theproc.returncode > 0:
+				print "Failed to start mom "
+				sys.exit(11)
+	# Start comm
+	with open(os.devnull, 'w') as fd:
+			theproc= subprocess.Popen(["sc", "start", "pbs_comm"], stdout=fd, stderr=fd)
+			theproc.communicate()
+			if theproc.returncode > 0:
+				print "Failed to start comm "
+				sys.exit(11)
+	# Start server
+	'''
+	with open(os.devnull, 'w') as fd:
+			theproc= subprocess.Popen(["sc", "start", "pbs_server"], stdout=fd, stderr=fd)
+			theproc.communicate()
+			if theproc.returncode > 0:
+				print "Failed to start server "
+				sys.exit(11)
+		
+	'''
+def main(argv):
+   installtype = 'server'
+   username = ''
+   password = ''
+   try:
+      opts, args = getopt.getopt(argv,"ht:u:p:",["type=","user=","passwd="])
+   except getopt.GetoptError:
+      print 'post_install.py -t <type> -u <username> -p <password>'
+      sys.exit(2)
+   for opt, arg in opts:
+      if opt == '-h':
+         print 'post_install.py -t <type> -u <username> -p <password>'
+         sys.exit()
+      elif opt in ("-t", "--type"):
+         installtype = arg
+      elif opt in ("-u", "--user"):
+         username= arg
+      elif opt in ("-p", "--passwd"):
+         password= arg
+   print 'Type of installation :', installtype 
+   print 'Username :', username
+   print 'Password :', password
+   var=validate_credentials(username, password)
+   createPbsConfFile()
+   installing_vcredist()
+   InstallDataService(username, password)
+   run_pbsmkdirs()
+   RegistingAndStartingServices(username, password)
+   print "Successfully installed"
+
+if __name__ == "__main__":
+   install_path= os.path.dirname(os.path.realpath(__file__))
+   pbs_path= install_path[:-8]
+   main(sys.argv[1:])
```

Comment:
Are these files written in DOS mode? That's often the case when I see no carriage return on the last line.